deploy/liquityStrategy.ts

```diff
@@ -58,12 +60,74 @@ const func = async function (env: HardhatRuntimeEnvironment) {
       args: [],
     });
 
-    await deploy('LiquityStabilityPool', {
+    const liquityStabilityPoolArgs = [
+      LUSDDeployment.address,
+      mockLiquityPriceFeedDeployment.address,
+    ];
+    const mockLiquityStabilityPool = await deploy('LiquityStabilityPool', {
       contract: 'MockStabilityPool',
       from: deployer,
-      args: [LUSDDeployment.address, mockLiquityPriceFeedDeployment.address],
+      args: liquityStabilityPoolArgs,
+      log: true,
+    });
+
+    const troveManagerDeploymentArgs = [
+      mockLiquityStabilityPool.address,
+      mockLiquityPriceFeedDeployment.address,
+    ];
+    const troveManagerDeployment = await deploy('TroveManager', {
+      contract: 'MockTroveManager',
+      from: deployer,
+      log: true,
+      args: troveManagerDeploymentArgs,
+    });
+
+    const mock0x = await deploy('Mock0x', {
+      contract: 'Mock0x',
+      from: deployer,
+      args: [],
       log: true,
     });
+
+    address0x = mock0x.address;
+
+    if (getNetworkName() !== 'hardhat' && getNetworkName() !== 'docker') {
+      try {
+        env.run('verify:verify', {
+          address: mockLiquityPriceFeedDeployment.address,
+          constructorArguments: [],
+        });
+      } catch (e) {
+        console.error((e as Error).message);
+      }
+
+      try {
+        env.run('verify:verify', {
+          address: troveManagerDeployment.address,
+          constructorArguments: troveManagerDeploymentArgs,
+        });
+      } catch (e) {
+        console.error((e as Error).message);
+      }
+
+      try {
+        env.run('verify:verify', {
+          address: mockLiquityStabilityPool.address,
+          constructorArguments: liquityStabilityPoolArgs,
+        });
+      } catch (e) {
+        console.error((e as Error).message);
+      }
+
+      try {
+        env.run('verify:verify', {
+          address: mock0x.address,
+          constructorArguments: [],
+        });
+      } catch (e) {
+        console.error((e as Error).message);
+      }
```

Comment:
You removed all the `await`, so as far as I can tell, you made these try/catch useless since they will never catch and log errors. So we'll never know if these fail and we have to re-run.